src/ngraph/frontend/onnx_import/core/graph.cpp

```diff
@@ -93,11 +98,39 @@ namespace ngraph
         } // namespace detail
 
         Graph::Graph(const onnx::GraphProto& graph_proto, Model& model, const Weights& weights)
-            : m_graph_proto{&graph_proto}
+            : m_graph_proto{graph_proto}
             , m_model{&model}
         {
+            const onnx::OpSchemaRegistry* schema_registry = onnx::OpSchemaRegistry::Instance();
+            for (const auto& node : m_graph_proto.node())
+            {
+                const auto node_op_schema = schema_registry->GetSchema(
+                    node.op_type(), static_cast<int>(ONNX_OPSET_VERSION), node.domain());
+
+                if (node_op_schema && node_op_schema->HasFunction())
+                {
+                    const onnx::FunctionProto* proto_func = node_op_schema->GetFunction();
+                    onnx::FunctionExpandHelper(node, *proto_func, m_graph_proto);
+                }
+                else
+                {
+                    std::cout << "has no function" << std::endl;
+                }
+            }
+
+            // DEBUG
+            for (const auto& node : m_graph_proto.node())
+            {
+                std::cout << node.op_type() << std::endl;
+            }
+            std::string data;
+            m_graph_proto.SerializeToString(&data);
+            std::cout << data;
+
+            // END DEBUG
```

Comment:
This code should be removed before making a final PR.

Reply:
Done